bower.json

```diff
@@ -17,7 +17,7 @@
     "animate.css": "~3.4.0",
     "clipboard": "~1.5.5",
     "jquery.scrollTo": "~2.1.2",
-    "patternfly": "3.6.0",
+    "patternfly": "f012b78c7f90fada15718705a6d99f72d7060d5c",
     "wowjs": "~1.1.2"
```

Comment:
When we cut a new release, at the end of the sprint, this will be replaced with 3.6.1 or whatever tag we decide to create